web/profiles/custom/os2loop/themes/os2loop_theme/assets/comment.scss

```diff
@@ -0,0 +1,36 @@
+/**
+ * @file
+ * Basic styling for comment module.
+ */
+
+/**
+ * Indent threaded comments.
+ */
+
+// Only indent within post/question, to avoid indentation in the reply page.
+.os2loop-post,
+.os2loop-question {
+  .indented {
+    margin-left: 25px;
+    margin-top: -20px;
+  }
+
+  .top-comment {
+    background-color: $primary-color;
+    border-radius: 0;
+  }
+
+  .top-comment-background {
+    background-color: $primary-color;
+    padding: 2em;
+    margin-bottom: 2em;
```

Comment:
Would it be better to use Bootstrap's `$spacer` variable here (cf. https://github.com/twbs/bootstrap/blob/v4.6.0/scss/_variables.scss#L136)?